frontend/src/stores/upload.ts

```diff
@@ -168,7 +159,7 @@ export const useUploadStore = defineStore("upload", {
       this.processUploads();
     },
     finishUpload(item: UploadItem) {
-      this.setProgress({ id: item.id, loaded: item.file.size > 0 });
+      this.setProgress({ id: item.id, loaded: item.file.size });
```

Comment:
The upload size was being discarded after finishing, making the total progress to go backwards. The size will now be retained, allowing the total progress to be properly calculated.